### ISSUE TITLE
Either template or templateFile must be set

### ISSUE THREAD

**Issue by retorquere** [nickdeis/eslint-plugin-notice#9]
When I use the first example from the readme:

```
"notice/notice":["error",{"mustMatch":"Copyright \\(c\\) [0-9]{0,4}, Nick Deis"}]
```

I get an error:

```
Either template or templateFile must be set
```

**Comment by nickdeis** [comment]
Hey @retorquere,
Thank you for filling an issue. This seems like a defect, since the expected behavior is that it checks headers/notices without a template file. I should be able to fix this in the next few days.
Best,
Nick

**Comment by nickdeis** [comment]
Hey @retorquere,
I have implemented the fix in version 0.7.8. Once again, thanks for pointing this issue out to me! I hope the fix works out for you, and please let me know if there is anything else I can help you with.
Cheers,
Nick
